hardhat.config.ts

```diff
@@ -150,7 +150,21 @@ const config: HardhatUserConfig = {
     ],
   },
   networks: {
-    hardhat: isFork(),
+    hardhat: {
+      allowUnlimitedContractSize: true,
+      loggingEnabled: false,
+      live: false,
+    },
+    mainnetfork: {
+      allowUnlimitedContractSize: false,
+      loggingEnabled: false,
+      url: process.env.FORK_MAINNET_RPC as string,
+      forking: {
+        url: process.env.FORK_MAINNET_RPC as string,
+        blockNumber: 21068448,
```

Comment:
I guess we could use an env var here as well?

It's really sad hardhat config can't accept parameters based on the test we're running. It would be nice if we could specify the fork block right in the test suite – but afaik hh doesn't support this because it doesn't expose the network setup to tests.

Reply:
sounds good